### ISSUE TITLE
Go DS SDK encode the float as base64 string by little-endian binary

### ISSUE THREAD

**Issue by weichou1229** [edgexfoundry/device-sdk-go#457]
Modify the float base64 string encoding function. 
Convert binary from big-endian to little-endian and then encode the float value to base64 string.

**Proposal:**
Change the ValueToString function
1. Convert cv.NumericValue to little-endian binary
2. Encode the binary to base64 string

https://github.com/edgexfoundry/device-sdk-go/blob/6b5c10e5a815794950d6808b6cccf5c95c2b64e4/pkg/models/commandvalue.go#L340
https://github.com/edgexfoundry/device-sdk-go/blob/6b5c10e5a815794950d6808b6cccf5c95c2b64e4/pkg/models/commandvalue.go#L350

**Reference:**
This modification is base on the DS SDK spec:
> When EdgeX is given (or returns) a float32 or float64 value as a string, the format of the string is by default a base64 encoded little-endian of the float32 or float64 value

https://docs.google.com/document/d/1aMIQ0kb46VE5eeCpDlaTg8PP29-DBSBTlgeWrv6LuYk/edit

https://play.golang.org/p/gbQ8NpRPaHO

**Comment by cloudxxx8** [comment]
this change will break the backward compatibility, so reopen it
we should fix it in 2.0

**Comment by tonyespy** [comment]
We just discussed this in the devices WG, it turns out the spec was changed in Edinburgh from big endian to little endian. The C SDK was changed, but the Go SDK wasn't, so all floats are converted to big endian before encoding, hence the spec mismatch.

We can't change this in a 1.x version of device-sdk-go as it would break existing applications.

The current consensus is that we should probably just drop base64 encoding for v2. As such, we should make sure this gets brought up as part of our Ireland planning.
